src/controllers/controls.js

```diff
@@ -18,11 +18,15 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
```

Comment:
Does this mean we no longer provide a base class for non-mapbox controls?

Reply:
The only difference between the two was in panToRotate. Both implementations are present here, in this incarnation, it autodetects the state type.

Reply:
What about orbit controller?

Reply:
> What about orbit controller?

It should be in there. Controls is essentially a unification of `OrbitControls` and `MapControls` that also works as `FirstPersonControls`. It is still possible to subclass controller and add one to the `ViewportController` but not necessary in typical cases.

Let me go through all the examples using `OrbitController` and make sure they still work, will update.